docs/en/reference/dql-doctrine-query-language.rst

```diff
@@ -672,18 +672,18 @@ The same restrictions apply for the reference of related entities.
 
     DQL DELETE statements are ported directly into an SQL DELETE statement.
     Therefore, some limitations apply:
-    
+
     - Lifecycle events for the affected entities are not executed.
-    - A cascading ``remove`` operation (as indicated e. g. by ``cascade={"remove"}`` 
-      or ``cascade={"all"}`` in the mapping configuration) is not being performed 
+    - A cascading ``remove`` operation (as indicated e. g. by ``cascade={"remove"}``
+      or ``cascade={"all"}`` in the mapping configuration) is not being performed
```

Comment:
This is annotations syntax and should be changed to attributes syntax as well. We should scan the docs for more occurrences.

```suggestion
    - A cascading ``remove`` operation (as indicated e. g. by ``cascade: ['remove']``
      or ``cascade: ['all']`` in the mapping configuration) is not being performed
```